sentry/src/main/java/io/sentry/Scope.java

```diff
@@ -495,6 +522,7 @@ public void removeContexts(final @NotNull String key) {
     clone.extra = extraClone;
 
     clone.contexts = contexts.clone();
+    clone.attachments = new CopyOnWriteArrayList<>(attachments);
```

Comment:
M: We might need to clone the Objects too, we do with breadcrumbs

Reply:
Makes sense